src/util/builtinOperators.ts

```diff
@@ -272,4 +275,41 @@ function generateGFunction(rm: rbac.RoleManager): any {
   };
 }
 
-export { keyMatchFunc, keyMatch2Func, keyMatch3Func, regexMatchFunc, ipMatchFunc, generateGFunction, keyMatch4Func, globMatch };
+function generateSyncGFunction(rm: rbac.RoleManager): syncGFuntion {
+  const memorized = new Map<string, boolean>();
```

Comment:
Check whether rm is async or sync.

Reply:
This has been checked at https://github.com/casbin/casbin.js/pull/150/files#diff-87ea86a52d699e299d03c1fa17d92615bcccbc48ffb360e3b8d8662e1c8af3f4R413